test/run/ch-run_misc.bats

```diff
@@ -1009,3 +1009,30 @@ EOF
     [[ $output = *"UID ${uid_bad} not found; using dummy info"* ]]
     [[ $output = *"GID ${gid_bad} not found; using dummy info"* ]]
 }
+
+@test 'Docker metadata' {
+    scope standard
+    [[ $CH_BUILDER = docker ]] || skip 'Docker only'
+    image="centos7"
+    diff -u - "$ch_imgdir/$image/ch/metadata.json" <<'EOF'
+{
+  "arch": "amd64",
+  "cwd": "",
+  "env": {
+    "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+  },
+  "labels": {
+    "org.label-schema.build-date": "20191001",
```

Comment:
This date likely gives a brittle diff. Jordan taught me about `diff -I` which could be used to exclude the date from comparison.

Reply:
Ooh that's nifty. I'll check it out.